packages/routing-interface/util/RIFUtil.ts

```diff
@@ -43,7 +43,7 @@ export class RIFUtil {
             return JSON.parse(jsonStr);
         }
         catch (error) {
-            logger.error(error)
+            // logger.error(error)
```

Comment:
@takeutak 
I recommend disambiguation between legit JSON parsing errors and the case of the try block just running 

```typescript
JSON.parse("null")
```

which will return null the same way the catch block will. Currently there's no way for the caller to find out which one has happened.
With the logging commented out even if they are looking at the logs they won't be able to tell the difference.

Reply:
Thank you for your comment.  I corrected this file as your comment.
https://github.com/hyperledger/cactus/pull/442/commits/7c88afda62b140e9573f920a313e953215408058